scripts/get_users_meta_docs.js

```diff
@@ -0,0 +1,102 @@
+const inquirer = require('inquirer'),
+      PouchDB = require('pouchdb-core'),
+      fs = require('fs'),
+      path = require('path'),
+      minimist = require('minimist');
+
+PouchDB.plugin(require('pouchdb-adapter-http'));
+
+const argv = minimist(process.argv.slice(2));
+
+if(argv['h'] || argv['help']) {
```

Comment:
NB:

```suggestion
if(argv.h || argv.help) {
```

Is valid, and is normally enforced by eslint, so you should probably use that. It's an oversight on our part that our scripts directory isn't yet covered, I'll do that soon.

Reply:
NB: if you want to can create aliases with minimist, so you can map `h` and `help` to the same thing. It would be something like:

```
const argv = minimist(process.argv.slice(2), {
  alias: {
    'h': 'help'
  }
});